overlord/configstate/configcore/refresh.go

```diff
@@ -37,6 +37,7 @@ func init() {
 	supportedConfigurations["core.refresh.metered"] = true
 	supportedConfigurations["core.refresh.retain"] = true
 	supportedConfigurations["core.refresh.rate-limit"] = true
+	supportedConfigurations["core.refresh.max-inhibition-days"] = true
```

Comment:
the name probably needs an ACK from @pedronis

Reply:
the name looks reasonable all things considered